src/lib/server/db/reports.channels.posts.sql.ts

```diff
@@ -0,0 +1,23 @@
+import { pgTable, text, uuid } from 'drizzle-orm/pg-core';
+import { post } from './posts.sql';
+import { channel } from './channels.sql';
+
+export const channelPostReport = pgTable('channel_post_report', {
+    id: uuid('id').primaryKey().defaultRandom(),
+    postId: uuid('post_id')
+        .notNull()
+        .references(() => post.id),
+    channelId: uuid('channel_id')
+        .notNull()
+        .references(() => channel.id),
+    description: text('description').notNull(),
+});
```

Comment:
Note here, but also for the other reports:
- Would be nice to have a "resolution" or "notes" column here for the admin/moderator to put some sort of message about how the report was handled
- Would also be nice to have a column to note the state of the report, i.e., "New", "Review", "Closed"

Reply:
might be better to just a table that relates to this, no?

Reply:
What would the relation be?

Reply:
I suppose we could treat it similar to GitHub issues where there are comments related to a report (an Issue on github). However, I think having a top-level *resolution* would still be nice. I'll add that.